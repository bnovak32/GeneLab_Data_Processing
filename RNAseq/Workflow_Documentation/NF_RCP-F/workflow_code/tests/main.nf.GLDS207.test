nextflow_pipeline {

    name "Test Workflow main.nf"
    script "main.nf"

    test("GLDS-207:Drosophila melanogaster,PairedEnd,ERCC: Should run without failures") {

        when {
            params {
                gldsAccession = 'GLDS-207' // GeneLab Data Accession Number, e.g. GLDS-104
                use_dummy_gene_counts = true // Use random gene counts for Deseq2, this addresses an issue where low/zero gene counts causes DGE analysis to fail
                truncateTo = 300 // Subsamples the raw reads files to the specified number of reads for EACH raw reads file.
            }
        }

        then {
            def TEST_DATASET = "GLDS-207"
            def TEST_SAMPLE_1 = "KCNQ370_Female_Ground_Control-02"
            def TEST_SAMPLE_2 = "Sei_ts1-Female_Ground_Control-09"
            def TEST_SAMPLE_3 = "KCNQ370_Female_Space_Flown-17"
            def TEST_SAMPLE_4 = "KCNQ97_Female_Space_Flown-21"
            def TEST_SAMPLE_5 = "Sei_ts1-Female_Space_Flown-25"
            assert workflow.success
            assert snapshot(
                            path("${params.outputDir}/${ TEST_DATASET }/Metadata").list(),
                            path("${params.outputDir}/${ TEST_DATASET }/00-RawData/Fastq").list(),
                            path("${params.outputDir}/${ TEST_DATASET }/01-TG_Preproc/Fastq").list(),
                            path("${params.outputDir}/${ TEST_DATASET }/02-STAR_Alignment/${ TEST_SAMPLE_1 }/${ TEST_SAMPLE_1 }_Aligned.sortedByCoord_sorted.out.bam"),
                            path("${params.outputDir}/${ TEST_DATASET }/02-STAR_Alignment/STAR_NumNonZeroGenes.csv"),
                            path("${params.outputDir}/${ TEST_DATASET }/02-STAR_Alignment/STAR_Unnormalized_Counts.csv"),
                            path("${params.outputDir}/${ TEST_DATASET }/03-RSEM_Counts/${ TEST_SAMPLE_2 }.genes.results"),
                            path("${params.outputDir}/${ TEST_DATASET }/03-RSEM_Counts/${ TEST_SAMPLE_2 }.isoforms.results"),
                            path("${params.outputDir}/${ TEST_DATASET }/03-RSEM_Counts/RSEM_NumNonZeroGenes.csv"),
                            path("${params.outputDir}/${ TEST_DATASET }/03-RSEM_Counts/RSEM_Unnormalized_Counts.csv"),
                            path("${params.outputDir}/${ TEST_DATASET }/04-DESeq2_NormCounts").list(),
                            path("${params.outputDir}/${ TEST_DATASET }/05-DESeq2_DGE").list(),
                            path("${params.outputDir}/${ TEST_DATASET }/GeneLab").list(),
                            path("${params.outputDir}/${ TEST_DATASET }/RSeQC_Analyses/02_geneBody_coverage/${ TEST_SAMPLE_3 }/${ TEST_SAMPLE_3 }.geneBodyCoverage.txt"),
                            path("${params.outputDir}/${ TEST_DATASET }/RSeQC_Analyses/03_infer_experiment/${ TEST_SAMPLE_3 }_infer_expt.out"),
                            // path("${params.outputDir}/${ TEST_DATASET }/RSeQC_Analyses/04_inner_distance/${ TEST_SAMPLE_4 }/${ TEST_SAMPLE_4 }.inner_distance.txt"),
                            path("${params.outputDir}/${ TEST_DATASET }/RSeQC_Analyses/05_read_distribution/${ TEST_SAMPLE_5 }_read_dist.out"),
                            ).match()
        }
    }
}
