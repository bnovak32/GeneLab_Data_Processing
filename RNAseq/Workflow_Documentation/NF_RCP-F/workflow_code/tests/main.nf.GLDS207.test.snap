{
    "GLDS-207:Drosophila melanogaster,PairedEnd,ERCC: Should run without failures": {
        "content": [
            [
                "GLDS-207_bulkRNASeq_v1_runsheet.csv:md5,1d17e342a765f2aca04db07d1916ae41",
                "GLDS-207_metadata_GLDS-207-ISA.zip:md5,8c0b6b689685ea8a1207187aa2a064c5",
                "GLDS-207_metadata_table.txt:md5,529d7d803db01be2675f1305a24110a5"
            ],
            [
                "CS-Female_Ground_Control-13_R1_raw.fastq.gz:md5,bff0a8fba9bb0ef1a22074b154b33d5d",
                "CS-Female_Ground_Control-13_R2_raw.fastq.gz:md5,02add22372bd2049d77dbcb3a77973c0",
                "CS-Female_Ground_Control-14_R1_raw.fastq.gz:md5,91ba5ce9544aaf6dfb9f6c465634cad3",
                "CS-Female_Ground_Control-14_R2_raw.fastq.gz:md5,b1f0fc37149ea527649f6ade27c26ecc",
                "CS-Female_Ground_Control-15_R1_raw.fastq.gz:md5,039fea6f3c0a37c9ff2ea7e762951484",
                "CS-Female_Ground_Control-15_R2_raw.fastq.gz:md5,e5cbb5ed8dfa2ac2bf7dfc4ef4f874b3",
                "CS-Female_Ground_Control-16_R1_raw.fastq.gz:md5,01769825c5e502d289397b36f2070964",
                "CS-Female_Ground_Control-16_R2_raw.fastq.gz:md5,fc4562d53e2c66720e608707512fcad0",
                "CS-Female_Space_Flown-29_R1_raw.fastq.gz:md5,37894e146c46c25ba804e782fc8c64e2",
                "CS-Female_Space_Flown-29_R2_raw.fastq.gz:md5,706496c8f3ba81e77783474eecac5d4d",
                "CS-Female_Space_Flown-30_R1_raw.fastq.gz:md5,1e01dd6a882d56ba61aa4fd34f943414",
                "CS-Female_Space_Flown-30_R2_raw.fastq.gz:md5,f0e52080fabfed7852a6e19692a3d71f",
                "CS-Female_Space_Flown-31_R1_raw.fastq.gz:md5,9433d7edf00c281431dd598819741995",
                "CS-Female_Space_Flown-31_R2_raw.fastq.gz:md5,83cc7b77efd1d60d3a13a91608e5600d",
                "CS-Female_Space_Flown-32_R1_raw.fastq.gz:md5,312a7f65eabf14e4750c0f908bd19f36",
                "CS-Female_Space_Flown-32_R2_raw.fastq.gz:md5,a6eccc469034fdf404d8ab48f0310127",
                "KCNQ370_Female_Ground_Control-01_R1_raw.fastq.gz:md5,443798ec947168c8c16d5eb0d4c77725",
                "KCNQ370_Female_Ground_Control-01_R2_raw.fastq.gz:md5,ea62136709d172d82a53aa9e7430f8a9",
                "KCNQ370_Female_Ground_Control-02_R1_raw.fastq.gz:md5,4c4ef0738b26753ebeda3a87faec813a",
                "KCNQ370_Female_Ground_Control-02_R2_raw.fastq.gz:md5,3b4008ef404fed72705994d713fb1822",
                "KCNQ370_Female_Ground_Control-03_R1_raw.fastq.gz:md5,e342161ee8591b9889368bb6e9fb528b",
                "KCNQ370_Female_Ground_Control-03_R2_raw.fastq.gz:md5,3224c66757192dbfc7713df8600063a2",
                "KCNQ370_Female_Ground_Control-04_R1_raw.fastq.gz:md5,4145d67d9102bca80f10f10317066e6d",
                "KCNQ370_Female_Ground_Control-04_R2_raw.fastq.gz:md5,10576db54691e4a4c016016b63be53c0",
                "KCNQ370_Female_Space_Flown-17_R1_raw.fastq.gz:md5,a1c275b20842193ba404086fa6285aa8",
                "KCNQ370_Female_Space_Flown-17_R2_raw.fastq.gz:md5,bc09d9d6da10634f0f0cd337d2ca3136",
                "KCNQ370_Female_Space_Flown-18_R1_raw.fastq.gz:md5,93bccb1a9e8bd6aee90b854da576ba0e",
                "KCNQ370_Female_Space_Flown-18_R2_raw.fastq.gz:md5,10d5632d990e54301c05bcb1991c27d7",
                "KCNQ370_Female_Space_Flown-19_R1_raw.fastq.gz:md5,6be088e8e459e9938fb11ba7cf637124",
                "KCNQ370_Female_Space_Flown-19_R2_raw.fastq.gz:md5,cb5127af3bece1d58f08cc9e2dd3c908",
                "KCNQ370_Female_Space_Flown-20_R1_raw.fastq.gz:md5,31f74aa91fd6702c6505bac9bed341c0",
                "KCNQ370_Female_Space_Flown-20_R2_raw.fastq.gz:md5,27f28545c29b8a8c6c22ce31c0e5d041",
                "KCNQ97_Female_Ground_Control-05_R1_raw.fastq.gz:md5,5aa9d8477c1841c1f9af95eb5711f3f7",
                "KCNQ97_Female_Ground_Control-05_R2_raw.fastq.gz:md5,bac85e459821b3e2707b0ac150f41e62",
                "KCNQ97_Female_Ground_Control-06_R1_raw.fastq.gz:md5,6cc934c675ffac3e3dd9d4316935f705",
                "KCNQ97_Female_Ground_Control-06_R2_raw.fastq.gz:md5,1671529921a0a7f795d66851d17c9778",
                "KCNQ97_Female_Ground_Control-07_R1_raw.fastq.gz:md5,584a7653f2a306f72e6998a19194d13d",
                "KCNQ97_Female_Ground_Control-07_R2_raw.fastq.gz:md5,c068c0f26de5f41bad290778bb93372e",
                "KCNQ97_Female_Ground_Control-08_R1_raw.fastq.gz:md5,26694db0008a5c062ede1afcfbc2a28f",
                "KCNQ97_Female_Ground_Control-08_R2_raw.fastq.gz:md5,19e026b2f15555c3a864f89eea4ae15e",
                "KCNQ97_Female_Space_Flown-21_R1_raw.fastq.gz:md5,ac9862be78b3ec98c2c8a6bbfa242315",
                "KCNQ97_Female_Space_Flown-21_R2_raw.fastq.gz:md5,b2e2bd78b7fa1e07c62d982340d0ce11",
                "KCNQ97_Female_Space_Flown-22_R1_raw.fastq.gz:md5,af3ed3634f6a9558db62744c612c5a4f",
                "KCNQ97_Female_Space_Flown-22_R2_raw.fastq.gz:md5,ad8c6ecc9c719075544bc00765022562",
                "KCNQ97_Female_Space_Flown-23_R1_raw.fastq.gz:md5,b7396380b3b4cd78df58a43206eaec98",
                "KCNQ97_Female_Space_Flown-23_R2_raw.fastq.gz:md5,87fc2a75507425156c3b41cd9bfaccaf",
                "KCNQ97_Female_Space_Flown-24_R1_raw.fastq.gz:md5,6e1356acec95a555ab1bc6e300bf8108",
                "KCNQ97_Female_Space_Flown-24_R2_raw.fastq.gz:md5,11b007dce8ba324beb02d82343c547d8",
                "Sei_ts1-Female_Ground_Control-09_R1_raw.fastq.gz:md5,c69056cedf8bd4c0adc8d2cb068ffa5e",
                "Sei_ts1-Female_Ground_Control-09_R2_raw.fastq.gz:md5,34e34b43f1fcdfed2cbe2937528236bb",
                "Sei_ts1-Female_Ground_Control-10_R1_raw.fastq.gz:md5,d8254908c3e2a9522b8fda8f53e137f4",
                "Sei_ts1-Female_Ground_Control-10_R2_raw.fastq.gz:md5,81b9a9a049b91d5cd540249fa8a90fe2",
                "Sei_ts1-Female_Ground_Control-11_R1_raw.fastq.gz:md5,ff983825ce08db637c8e9f719d158754",
                "Sei_ts1-Female_Ground_Control-11_R2_raw.fastq.gz:md5,e1025aeaf4a1dab3fe7bdf52ae87a518",
                "Sei_ts1-Female_Ground_Control-12_R1_raw.fastq.gz:md5,1ebfd528073bd2dbe5827540f6693669",
                "Sei_ts1-Female_Ground_Control-12_R2_raw.fastq.gz:md5,e9bd880cb001c569b54b431cd01db1df",
                "Sei_ts1-Female_Space_Flown-25_R1_raw.fastq.gz:md5,9a46474adbcd153ae2bce82d74498f9a",
                "Sei_ts1-Female_Space_Flown-25_R2_raw.fastq.gz:md5,4162f509cd98452a01cb6b9c88c4af9e",
                "Sei_ts1-Female_Space_Flown-26_R1_raw.fastq.gz:md5,4de4d970bd76e7ce570d3b9b4cd04eef",
                "Sei_ts1-Female_Space_Flown-26_R2_raw.fastq.gz:md5,8bb9783b9b8b1509fa0ecc79c3679ac4",
                "Sei_ts1-Female_Space_Flown-27_R1_raw.fastq.gz:md5,16ae642e9ad797f941084fef1508f6ba",
                "Sei_ts1-Female_Space_Flown-27_R2_raw.fastq.gz:md5,e274e315c093cd38aa1d80bf05ddf11f",
                "Sei_ts1-Female_Space_Flown-28_R1_raw.fastq.gz:md5,5ab0c7ac4cfe4df331b12456c3e88c5e",
                "Sei_ts1-Female_Space_Flown-28_R2_raw.fastq.gz:md5,513327e0892f2304232ab6b122d9cb34"
            ],
            [
                "CS-Female_Ground_Control-13_R1_trimmed.fastq.gz:md5,b034a9c295080ae3f991797d48cf660b",
                "CS-Female_Ground_Control-13_R2_trimmed.fastq.gz:md5,1b12f21fa59bc9b35c174147ef6bcd99",
                "CS-Female_Ground_Control-14_R1_trimmed.fastq.gz:md5,30de5efe4f8f20a6b3c559647e66cc14",
                "CS-Female_Ground_Control-14_R2_trimmed.fastq.gz:md5,40936527b519c5afd8acd8d6c85c0c6d",
                "CS-Female_Ground_Control-15_R1_trimmed.fastq.gz:md5,c458466b0a5bec1e4ec09b58df049780",
                "CS-Female_Ground_Control-15_R2_trimmed.fastq.gz:md5,9f1d636b670621d8a3ff7c5934fd5e4c",
                "CS-Female_Ground_Control-16_R1_trimmed.fastq.gz:md5,79f0b9f4b58b9fa50556d91c74851f72",
                "CS-Female_Ground_Control-16_R2_trimmed.fastq.gz:md5,13a5c53e45ebb199e427c439db0312ea",
                "CS-Female_Space_Flown-29_R1_trimmed.fastq.gz:md5,eaea077df048b9c04089cfa326a66c76",
                "CS-Female_Space_Flown-29_R2_trimmed.fastq.gz:md5,c9b7b1ba57a21e1449dcb93d766eaafb",
                "CS-Female_Space_Flown-30_R1_trimmed.fastq.gz:md5,eb8c3a6e562f81eb19eeab6d550d348f",
                "CS-Female_Space_Flown-30_R2_trimmed.fastq.gz:md5,525fbaf79a931ca4482f76cf9cb17f75",
                "CS-Female_Space_Flown-31_R1_trimmed.fastq.gz:md5,b00348a08f0bb949efe9bc84b424409e",
                "CS-Female_Space_Flown-31_R2_trimmed.fastq.gz:md5,13d785d84899a319db2ed6036b4a2273",
                "CS-Female_Space_Flown-32_R1_trimmed.fastq.gz:md5,bcd861ec117f9715550bda71391c7de6",
                "CS-Female_Space_Flown-32_R2_trimmed.fastq.gz:md5,72fb2cbfba84f64fe71d0adfd3ebd9ed",
                "KCNQ370_Female_Ground_Control-01_R1_trimmed.fastq.gz:md5,104b93397c6cd500cc382ec438e318cd",
                "KCNQ370_Female_Ground_Control-01_R2_trimmed.fastq.gz:md5,7ad80a849d228a83b38d9446a8be6439",
                "KCNQ370_Female_Ground_Control-02_R1_trimmed.fastq.gz:md5,6721e1346e3f7c1ffcab5c923878b5a0",
                "KCNQ370_Female_Ground_Control-02_R2_trimmed.fastq.gz:md5,b336bfe56335b5be3e13ec09902d5851",
                "KCNQ370_Female_Ground_Control-03_R1_trimmed.fastq.gz:md5,7608bdba317f92bbe7f0d241f25d04c5",
                "KCNQ370_Female_Ground_Control-03_R2_trimmed.fastq.gz:md5,ea51694c51c1fa522d09eef53651bab7",
                "KCNQ370_Female_Ground_Control-04_R1_trimmed.fastq.gz:md5,22d5ced375f7f5dd82594caa6b9f5ff5",
                "KCNQ370_Female_Ground_Control-04_R2_trimmed.fastq.gz:md5,70f409a09610e61a00ff752ca9f7247b",
                "KCNQ370_Female_Space_Flown-17_R1_trimmed.fastq.gz:md5,4e8e47daaf7df7a331e41a756f4c9508",
                "KCNQ370_Female_Space_Flown-17_R2_trimmed.fastq.gz:md5,b2ae0bb0336b513e9ce4d231369126b8",
                "KCNQ370_Female_Space_Flown-18_R1_trimmed.fastq.gz:md5,16ac867981a7ef35f8702f868e053298",
                "KCNQ370_Female_Space_Flown-18_R2_trimmed.fastq.gz:md5,054b85ebf548401b40fa1fab6d8590d8",
                "KCNQ370_Female_Space_Flown-19_R1_trimmed.fastq.gz:md5,966048358f88c33b8f2e3ef87a501681",
                "KCNQ370_Female_Space_Flown-19_R2_trimmed.fastq.gz:md5,c976626236c4bc4e903e289d7bfeaf69",
                "KCNQ370_Female_Space_Flown-20_R1_trimmed.fastq.gz:md5,6568cf24d732d9831119e5f8d5505181",
                "KCNQ370_Female_Space_Flown-20_R2_trimmed.fastq.gz:md5,a23bb25a19f4c8fc3cbec05285284a00",
                "KCNQ97_Female_Ground_Control-05_R1_trimmed.fastq.gz:md5,7edda8a2707d074469f0ea105e114873",
                "KCNQ97_Female_Ground_Control-05_R2_trimmed.fastq.gz:md5,031be9d6c6fc201b3c338de2d1b5f31c",
                "KCNQ97_Female_Ground_Control-06_R1_trimmed.fastq.gz:md5,c36596dbff548ea9e0db5e7ddb0c3caa",
                "KCNQ97_Female_Ground_Control-06_R2_trimmed.fastq.gz:md5,4cf066744322b014294a0ec7bab1cf03",
                "KCNQ97_Female_Ground_Control-07_R1_trimmed.fastq.gz:md5,525d7126c091a63c5c5628271fb160ca",
                "KCNQ97_Female_Ground_Control-07_R2_trimmed.fastq.gz:md5,3d9479da99d153e9722b20613d3511fc",
                "KCNQ97_Female_Ground_Control-08_R1_trimmed.fastq.gz:md5,21fcd48cc23931c26ca72c2f53433cf9",
                "KCNQ97_Female_Ground_Control-08_R2_trimmed.fastq.gz:md5,5153f5f178b53e4271c3ee669db62ac0",
                "KCNQ97_Female_Space_Flown-21_R1_trimmed.fastq.gz:md5,2e1de02f4995ffc391457f3ccdbd1f24",
                "KCNQ97_Female_Space_Flown-21_R2_trimmed.fastq.gz:md5,ce79c047583ff8e0cc5f74d972f3ff16",
                "KCNQ97_Female_Space_Flown-22_R1_trimmed.fastq.gz:md5,877ca60cd29df3c1a77813b37116f8bb",
                "KCNQ97_Female_Space_Flown-22_R2_trimmed.fastq.gz:md5,505608760452e2e4402180120a53a869",
                "KCNQ97_Female_Space_Flown-23_R1_trimmed.fastq.gz:md5,af08661b1f94d26a209db97366643748",
                "KCNQ97_Female_Space_Flown-23_R2_trimmed.fastq.gz:md5,aab2004abb2a307b723122b1d7539dfd",
                "KCNQ97_Female_Space_Flown-24_R1_trimmed.fastq.gz:md5,d9d2e3dabe5f736b681030cf4610e1e9",
                "KCNQ97_Female_Space_Flown-24_R2_trimmed.fastq.gz:md5,5f3a549607692b38c1e50f2af840dd85",
                "Sei_ts1-Female_Ground_Control-09_R1_trimmed.fastq.gz:md5,01caadb1d8c337800589ee169b40d670",
                "Sei_ts1-Female_Ground_Control-09_R2_trimmed.fastq.gz:md5,c8e7ef28d358f5a674592418fb662d3c",
                "Sei_ts1-Female_Ground_Control-10_R1_trimmed.fastq.gz:md5,1d087caa69f0368cac28e6a956ac25c9",
                "Sei_ts1-Female_Ground_Control-10_R2_trimmed.fastq.gz:md5,aa85406ab3f7fbb60d4f9169ac9cbde0",
                "Sei_ts1-Female_Ground_Control-11_R1_trimmed.fastq.gz:md5,402e110fe2f4ec4ab52177557c7ab6ef",
                "Sei_ts1-Female_Ground_Control-11_R2_trimmed.fastq.gz:md5,3123217b3529dd9c7379de4c5c41c52c",
                "Sei_ts1-Female_Ground_Control-12_R1_trimmed.fastq.gz:md5,1535fc36954f0ccff21f05ebb4f75274",
                "Sei_ts1-Female_Ground_Control-12_R2_trimmed.fastq.gz:md5,2d4b1774202e78ab9595ba713618bba0",
                "Sei_ts1-Female_Space_Flown-25_R1_trimmed.fastq.gz:md5,4072e82d429cd7159a60a393fedb9c23",
                "Sei_ts1-Female_Space_Flown-25_R2_trimmed.fastq.gz:md5,98ba2d413a73dff3a45b361e10a74b8c",
                "Sei_ts1-Female_Space_Flown-26_R1_trimmed.fastq.gz:md5,9d721028b959d3f0c17cfea7ac935c4d",
                "Sei_ts1-Female_Space_Flown-26_R2_trimmed.fastq.gz:md5,8cfa58f4029e988b36ab2cbbecc62d8e",
                "Sei_ts1-Female_Space_Flown-27_R1_trimmed.fastq.gz:md5,829105db2cdf9dbce40a00feae6cbf98",
                "Sei_ts1-Female_Space_Flown-27_R2_trimmed.fastq.gz:md5,72010bef7e89725a52a65758ecb3ebf1",
                "Sei_ts1-Female_Space_Flown-28_R1_trimmed.fastq.gz:md5,38e6e2f485110299809d9806b039aa4f",
                "Sei_ts1-Female_Space_Flown-28_R2_trimmed.fastq.gz:md5,adafc6484b64e23ee0e0084f1aa6d0c5"
            ],
            "KCNQ370_Female_Ground_Control-02_Aligned.sortedByCoord_sorted.out.bam:md5,f291b63201ee13da4683d3a6dbf471f1",
            "STAR_NumNonZeroGenes.csv:md5,563a9b561fae52ea0b8a1251ab2f2255",
            "STAR_Unnormalized_Counts.csv:md5,01726a25900d257fb8b3f84f3a1ffe0a",
            "Sei_ts1-Female_Ground_Control-09.genes.results:md5,c8509ad9d0ebbc8308bb14fc9fb85f42",
            "Sei_ts1-Female_Ground_Control-09.isoforms.results:md5,e025c4ce771cbab46aba4447d5ee29db",
            "RSEM_NumNonZeroGenes.csv:md5,e2f30fa09e112ec9389b69417cd2c017",
            "RSEM_Unnormalized_Counts.csv:md5,ae3624a2406b83a64c2f5815f2049dbd",
            [
                "ERCC_Normalized_Counts.csv:md5,a0ebd6f327dc088ee9528dfdaaa84550",
                "Normalized_Counts.csv:md5,53eeaf25b4f47918c3f6f0f905c4abba",
                "RSEM_Unnormalized_Counts.csv:md5,0b7ecd10bc101f40eb0939032d24e77f"
            ],
            [
                [
                    "ERCCnorm_SampleTable.csv:md5,0544bb6d9bd7eb68051cde7c740124de",
                    "ERCCnorm_contrasts.csv:md5,09eb850cb7edb77365d290f471d95b62",
                    "ERCCnorm_differential_expression.csv:md5,81cc27024c7d58275f2e3fba4eecd28e",
                    "visualization_PCA_table_ERCCnorm.csv:md5,fd55be29bece8114f98557d0a07d618e",
                    "visualization_output_table_ERCCnorm.csv:md5,4e1e6f4f9e29edf29541eda086d55a49"
                ],
                "SampleTable.csv:md5,0544bb6d9bd7eb68051cde7c740124de",
                "contrasts.csv:md5,09eb850cb7edb77365d290f471d95b62",
                "differential_expression.csv:md5,b78f0058a272b86c31fba2a2d0f7400a",
                "visualization_PCA_table.csv:md5,92a85b004968eb92619c5eea87c11b9f",
                "visualization_output_table.csv:md5,142adc46ed1be818e279135a57652210"
            ],
            [
                "software_versions.md:md5,9f59704a831ca5fba8383562d8e553d7"
            ],
            "KCNQ370_Female_Space_Flown-17.geneBodyCoverage.txt:md5,b38d64394c63c623eaedde6002cf44dc",
            "KCNQ370_Female_Space_Flown-17_infer_expt.out:md5,49b3812324e2d8d003713f53db03e50c",
            "Sei_ts1-Female_Space_Flown-25_read_dist.out:md5,22faf86e0232dce0a4d370ffeb0e0aa2"
        ],
        "timestamp": "2023-01-24T01:26:07+0000"
    }
}